helmfile/README.md

```diff
@@ -54,7 +80,7 @@ Should timeouts be experienced or if you noticed that some pods are experiencing
 it suggests that there are configuration errors.
 
 Some possible errors could be
-1. Blank fields in files found in `value`, if the field is not needed or optional. Remember to comment them out.
+1. Blank network providers found in `values/graphprotocol-node*` in `config.chains`, if the network wasn't configure don't remember to comment it out.
```

Comment:
Tbh, didn't fully understand what you meant here. Maybe the following line describes what you want to say:

```suggestion
Don't forget to fill in the `config.chains` value in the `values/graphprotocol-node*` file.
```
+
maybe `Some possible errors could be` line can be removed.